commands/utils/params.go

```diff
@@ -120,6 +121,11 @@ func extractGitParamsFromEnv(params *FrogbotParams) error {
 
 func extractGeneralParamsFromEnv(params *FrogbotParams) {
 	params.WorkingDirectory = getTrimmedEnv(WorkingDirectoryEnv)
+	var err error
+	params.IncludeAllVulnerabilities, err = strconv.ParseBool(getTrimmedEnv(IncludeAllVulnerabilitiesEnv))
+	if err != nil {
+		params.IncludeAllVulnerabilities = false
```

Comment:
Let's log the error, so customers can understand what went wrong with the value of this flag.